releasenotes/notes/change-trace-proxy-headers-f9fcbb4efc4fcb95.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    This change adds support for disabling / enabling proxying for a trace by sending 
+    the Datadog-Agent-Proxy-Enabled with a boolean value within a trace's headers. By default,
```

Comment:
Can we document this option in the README too?

Reply:
added